ElementX/Sources/Other/HTMLParsing/AttributedStringBuilder.swift

```diff
@@ -242,4 +242,8 @@ extension UIColor {
 extension NSAttributedString.Key {
     static let DTTextBlocks: NSAttributedString.Key = .init(rawValue: DTTextBlocksAttribute)
     static let MXBlockquote: NSAttributedString.Key = .init(rawValue: BlockquoteAttribute.name)
+    static let MXUserID: NSAttributedString.Key = .init(rawValue: UserIDAttribute.name)
+    static let MXRoomID: NSAttributedString.Key = .init(rawValue: RoomIDAttribute.name)
+    static let MXRoomAlias: NSAttributedString.Key = .init(rawValue: RoomAliasAttribute.name)
+    static let MXEventID: NSAttributedString.Key = .init(rawValue: EventIDAttribute.name)
```

Comment:
If these actually need to be prefixed (unsure if they do), then it seems like it would be better style to not shorten the prefix?
```suggestion
    static let MatrixBlockquote: NSAttributedString.Key = .init(rawValue: BlockquoteAttribute.name)
    static let MatrixUserID: NSAttributedString.Key = .init(rawValue: UserIDAttribute.name)
    static let MatrixRoomID: NSAttributedString.Key = .init(rawValue: RoomIDAttribute.name)
    static let MatrixRoomAlias: NSAttributedString.Key = .init(rawValue: RoomAliasAttribute.name)
    static let MatrixEventID: NSAttributedString.Key = .init(rawValue: EventIDAttribute.name)
```